src/components/PhotoChooser.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Box, Typography, useTheme } from '@mui/material';
+import { Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { useDeviceInfo } from '../utils';
+import ErrorBox from './ErrorBox/ErrorBox';
+import Button from './Button/Button';
+import Icon from './Icon/Icon';
```

Comment:
@constanzauanini I don't know how I missed this or how this even builds.. 
let's redo this please

Reply:
oops,, wrong place comment

Reply:
please ignore